Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
+    rack-test (2.0.2)
+      rack (>= 1.3)
+    rails (7.0.4)
+      actioncable (= 7.0.4)
+      actionmailbox (= 7.0.4)
+      actionmailer (= 7.0.4)
+      actionpack (= 7.0.4)
+      actiontext (= 7.0.4)
+      actionview (= 7.0.4)
+      activejob (= 7.0.4)
+      activemodel (= 7.0.4)
+      activerecord (= 7.0.4)
+      activestorage (= 7.0.4)
+      activesupport (= 7.0.4)
       bundler (>= 1.15.0)
-      railties (= 7.0.2.3)
+      railties (= 7.0.4)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.4.2)
+    rails-html-sanitizer (1.4.3)
       loofah (~> 2.3)
-    railties (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    railties (7.0.4)
+      actionpack (= 7.0.4)
+      activesupport (= 7.0.4)
       method_source
       rake (>= 12.2)
       thor (~> 1.0)
       zeitwerk (~> 2.5)
     rainbow (3.1.1)
     rake (13.0.6)
-    ransack (3.0.0)
-      activerecord (>= 6.0.4)
-      activesupport (>= 6.0.4)
+    ransack (3.2.1)
```

Comment:
### [ransack](https://github.com/activerecord-hackery/ransack)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.0 → 3.2.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/ransack/3.0.0/3.2.1)_]





<details>
<summary>Commits</summary>

A change of **42** commits. See the full changes on [the compare page](https://github.com/activerecord-hackery/ransack/compare/v3.0.0...v3.2.1).

These are the first 10 commits:
- (cfc727f) [Fix broken link](https://github.com/activerecord-hackery/ransack/commit/cfc727fa9545e1f00ec47604eeed5a42b3a08c7c)
- (f5f3a6e) [Remove npm things](https://github.com/activerecord-hackery/ransack/commit/f5f3a6e281843ee566918a9e337ecca153f3c029)
- (9588481) [Clarify documentation setup docs](https://github.com/activerecord-hackery/ransack/commit/9588481f93dae24a77e8bdebd46d44743f316244)
- (a7b2c27) [Merge pull request #1297 from activerecord-hackery/tweak-doc…](https://github.com/activerecord-hackery/ransack/commit/a7b2c27f1a22def7c1efecf18c43cade4c3dc82b)
- (02da46e) [Add list of wiki contributors](https://github.com/activerecord-hackery/ransack/commit/02da46e2f8e4428c23471b6b8c06bf6ba769649f)
- (ce75337) [Merge pull request #1300 from activerecord-hackery/wiki-cont…](https://github.com/activerecord-hackery/ransack/commit/ce753373c615adfa36aec310bf8a5badb2506f57)
- (0e120fc) [Workflows for automatic deployment of docs site (#1302)](https://github.com/activerecord-hackery/ransack/commit/0e120fc3cc03b3cc33f55bafa6d069e5de2b5f7b)
- (daa940e) [Fix documentation site edit urls (#1303)](https://github.com/activerecord-hackery/ransack/commit/daa940e9886e2797da1ef1731cec89ce97ca2265)
- (96e0708) [Don't pass `data` attribute appended to query params (#1301)](https://github.com/activerecord-hackery/ransack/commit/96e0708a72a54bf18f8ea6948e1868d4e195e34b)
- (5b9ee1d) [Add missing changelog entries](https://github.com/activerecord-hackery/ransack/commit/5b9ee1dd36ffe82910725324313a874828a8e85f)


</details>